tendrl/gluster_integration/manager/utils.py

```diff
@@ -4,24 +4,18 @@
 import subprocess
 
 LOG = logging.getLogger(__name__)
-TENDRL_CONTEXT = "/etc/tendrl/gluster_integration/tendrl_context"
+TENDRL_CONTEXT = "/var/lib/tendrl/gluster_cluster_id"
```

Comment:
the reason we were using "tendrl_context" was that it is a pre defined object in Tendrl

the new path "/var/lib/tendrl/gluster_cluster_id", how did you come up with this?